test/rpc-test.js

```diff
@@ -51,6 +51,9 @@ let txid = null;
 let utxo = null;
 
 describe('RPC', function() {
+  if (process.browser)
+    return;
```

Comment:
Looks like `this.skip()` is the preferred way of handling this in (b)mocha.
I know it's totally wacky but if you ran the command `npm run test-file-browser test/rpc-test.js` in this branch, bmocha would just hang forever. (and I don't know why but...) when switching this out for `this.skip()` bmocha will at least return with ` 0 passing (5ms), 19 pending`

Reply:
Good call on that!

Reply:
Going to address this in another PR